salt/modules/pkgng.py

```diff
@@ -296,52 +296,30 @@ def latest_version(*names, **kwargs):
     chroot = kwargs.get("chroot")
     refresh = kwargs.get("refresh")
     root = kwargs.get("root")
-    pkgs = list_pkgs(versions_as_list=True, jail=jail, chroot=chroot, root=root)
-
-    if salt.utils.versions.compare(
-        _get_pkgng_version(jail, chroot, root), ">=", "1.6.0"
-    ):
-        quiet = True
```

Comment:
Why are you removing quiet?

Reply:
If you look below, you'll see that I made the `-q` flag unconditional.  Previously, the only reason not to use it was to support pkg older than 1.6.0.  My commit 47fb63d drops support for that old pkg version.

Reply:
It looks like in a commit after that one you ended up removing `cmd.append("-q")` all together though. If I'm understanding correctly newer versions of pkg would need the `-q` flag, so shouldn't we keep it if we are only supporting newer versions?

Reply:
K I see it now. I was grepping for `-q`. You included it in `"-eqS"`. Thanks for helping my eyesight ;)